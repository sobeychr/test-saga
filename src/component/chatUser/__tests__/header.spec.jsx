import React from 'react';
import { shallow } from 'enzyme';
import { userMock } from 'Util/mock';
import toJson from 'enzyme-to-json';

import Header from './../header';

describe('src/component/charUser/header', () => {
    const {
        name: { display },
    } = userMock;

    it('should render properly', () => {
        const wrapper = shallow(<Header user={userMock} />);

        expect(wrapper).toHaveLength(1);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
