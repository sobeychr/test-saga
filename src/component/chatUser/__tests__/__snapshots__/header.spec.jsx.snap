// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/component/charUser/header should render properly 1`] = `
<div
  className="header"
>
  <Avatar
    user={
      Object {
        "avatar": 4,
        "gender": 0,
        "id": 1,
        "name": Object {
          "display": "_ut_nunc_h",
          "first": "uismod. Na",
          "last": "consectetu",
        },
      }
    }
  />
  <span>
    _ut_nunc_h
  </span>
</div>
`;
